tests/safeds_stubgen/__snapshots__/test_main.ambr

```diff
@@ -602,6 +602,21 @@
           'name': 'int',
         }),
       }),
+      dict({
+        'docstring': dict({
+          'default_value': '',
+          'description': '',
+          'type': '',
+        }),
+        'id': 'test_package/test_package/test_package/test_package/test_package',
```

Comment:
Is this intended?

Reply:
Removed these snapshot tests and added unit tests instead.